tests/src/test/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleTestHelper.scala

```diff
@@ -165,7 +165,7 @@ object RapidsShuffleTestHelper extends MockitoSugar with Arm {
   }
 
   def withMockContiguousTable[T](numRows: Long)(body: ContiguousTable => T): T = {
-    val rows: Seq[Integer] = (0 until numRows.toInt).map(new Integer(_))
+    val rows: Seq[Integer] = (0 until numRows.toInt).map(Int.box)
```

Comment:
nit:
```suggestion
    val rows: Seq[Integer] = (0 until numRows).map(Int.box)
```

Reply:
sorry, that was a bad comment, scalac won't do a silent lossy conversion for us. I'll undo it

Reply:
@gerashegalov no problem, the constructor (I was trying not to use) also came back with the revert, but I changed it back to `Int.box`. Please take another look!